content/projects/remoting/index.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: project
+title: "Jenkins Remoting"
+section: projects
+tags:
+- remoting
+- agent
+- agents
+- slave
+- slaves
+---
+
+Jenkins Remoting is a library and executable JAR, which implements communication layer in Jenkins: 
+TCP-based communication protocols, Remote procedure calls, classloading, data streaming, etc.
+It's being used for _master <=> agent_ communication and in several other cases (see below).
+
+:toc:
+
+== Remoting Usage in Jenkins
+
+* Agent executables (`slave.jar` is a renamed `remoting.jar`)
+* Master to Agent communication protocols being used in various Agent types, including Java Web Start (aka JNLP) and link:https://plugins.jenkins.io/ssh-slaves[SSH] agents 
+* link:/doc/book/managing/cli/[Jenkins CLI], Remoting mode is **deprecated since 2.54** (link:/blog/2017/04/11/new-cli/[announcement])
+* Communication with Maven instance in the link:https://plugins.jenkins.io/maven-plugin[Maven Integration plugin]
+
+== Project Components
+
+=== Remoting Library
+
+Remoting Library is a main component of the Remoting subproject.
+The code base is located in the link:https://github.com/jenkinsci/remoting[separate repository],
+which also contains Remoting developer and user documentation.
+This library also has an independent release cycle and LTS release lines.
+
+Links:
+
+* link:remotinglib[Home Page]
+* link:changelog[Release Notes]
+* link:protocols[Remoting Protocols]
+* link:troubleshooting[Troubleshooting] (under construction)
+
+=== Jenkins Core and Modules
+
+Although Remoting library is a standalone component, 
+Jenkins core includes some extension and modules for working with Remoting.
+It allows implementing custom protocols on-demand (e.g. for Jenkins CLI).
+There is ongoing activity on decoupling these components to plugins 
```

Comment:
"There is ongoing activity on detaching these core components to Jenkins plugins"